src/schema/me/follow_show.ts

```diff
@@ -0,0 +1,43 @@
+import { GraphQLString, GraphQLBoolean } from "graphql"
+import { mutationWithClientMutationId } from "graphql-relay"
+import { ShowType } from "../show"
+
+export default mutationWithClientMutationId({
+  name: "FollowShow",
+  description: "Follow (or unfollow) a show",
+  inputFields: {
+    partner_show_id: {
```

Comment:
Should be cameCase

Reply:
Other follow fields are not (including FollowProfile), because we're passing this in as a query parameter to gravity which expects snake case like this. 

https://github.com/artsy/metaphysics/blob/master/src/schema/me/follow_gene.ts#L9
https://github.com/artsy/metaphysics/blob/master/src/schema/me/follow_artist.ts#L10
https://github.com/artsy/metaphysics/blob/master/src/schema/me/follow_profile.ts#L9

I'm going to leave it snake case so that there aren't mismatches in the internals unless you feel strongly that it should be different from the others?

Reply:
@orta agrees that it should be camelCase not snake_case - but if we're going to aim for consistency I still think we should have a separate effort for that. If I change just shows now that's easy enough, but to change fairs too I'd have to change everything that consumes that schema etc and this PR feels like it's already done enough - plus then the codebase would be even more inconsistent, where different follow modules do different styles, etc.

Can we do an upkeep ticket for standardizing on variable naming, instead, and get this bad boy merged?

Reply:
If that happens soon enough then I'm ok with that. Keep in mind that schema changes _should_ ideally be backwards compatible, we definitely don't want somebody to start using this in the iOS app and deploying it before you'd change it, because then we no longer can.

Reply:
All new fields should be camelCase. Not sure what you’re referring to by other examples in this PR; if they are fields that already existed then they stay as they are.